### ISSUE TITLE
why long text prediction is so bad，and test accuracy is low？

### ISSUE THREAD

**Issue by garspace** [meijieru/crnn.pytorch#201]
environment: pytorch0.4.0  python3.7 
the model is trained on dataset my-self about 30w photos, lr=0.001, batchsize=16

**Comment by Mr-DDDAlKilanny** [comment]
I think CRNN is used mostly for recognizing a single word (e.g. max 10 characters per image). If you want you can use other model designed for that.
